### DISCUSSION TITLE
Ability to make buildings spawn items or animals

### DISCUSSION THREAD

**Discussion by Isus104** [TownyAdvanced/Towny#4630] [Ideas]
It would be amazing if mayors had the ability to make actually useful buildings in their towns.

For example, if someone makes an armory structure, they can then assign that building to be armory and then based on players in town (or something else) that armory can spawn weapons each hour.

Or farm which can spawn animals.

Or chuch which gives players inside of them specific buffs if they pray.. 

There is plenty of more possibilities with this but yeah basically that's it.

**Comment by LlmDl** [comment]
This is probably a pretty good idea for a separate plugin.

**Reply by Isus104** [reply]
Yeah i would love to see that and use in my server
